package-lock.json

```diff
@@ -11359,7 +11359,7 @@
 				},
 				"yargs": {
 					"version": "11.1.0",
-					"resolved": "http://registry.npmjs.org/yargs/-/yargs-11.1.0.tgz",
+					"resolved": "https://registry.npmjs.org/yargs/-/yargs-11.1.0.tgz",
```

Comment:
I think this URL changes may be unintentional? If they are intentional maybe we can do the update to https in a separate PR.

Reply:
@jorgefilipecosta I think that just happened on the npm install, but I can manually revert it if it's a concern